src/labels/CustomLabels.labels

```diff
@@ -2016,6 +2016,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Refresh Opportunities</shortDescription>
         <value>Refresh Opportunities</value>
     </labels>
+    <labels>
+        <fullName>RD_ErrorAddDonationLimitedToLegacyMode</fullName>
```

Comment:
@sam-knox , can you please verify the new custom label?
@meena-sfdc , this is a new label that once reviewed and approved by Sam here, can also be copied into the label freeze PR.

Reply:
Thanks, @RadSFDeveloper !